frontend/public/components/_catalog.scss

```diff
@@ -101,14 +101,24 @@ $catalog-item-icon-size-sm: 24px;
   &__num-items {
     border-bottom: 1px solid $color-grey-border;
     font-weight: 700;
-    margin: 0 ($grid-gutter-width / 2) $grid-gutter-width 0;
+    margin: 0 0 $grid-gutter-width 0;
     padding: 0 0 20px;
   }
 
   &__tabs {
     flex: 0 0 170px;
     margin: 0 ($grid-gutter-width / 2) 0 0;
   }
+
+  &__no-filter-results {
+    background-color: transparent;
+    border: none;
+    padding: 0;
+    &--title {
```

Comment:
This isn't really a modifier, it's a different element. Naming looks wrong. Maybe use

```
co-catalog-page__no-filter-results-title
```